### ISSUE TITLE
Link Title Attribute not displaying

### ISSUE THREAD

**Issue by tlovering** [GavickPro/Music-Free-Responsive-Joomla-Template#55]
I'm trying to add Menu sub-texts to my site, as seen in the Music demo. I'm assuming this is driven by the Link Title Attribute field in Link Type Options, but when I add text here it doesn't display.

I previously had S5 Flex Menu installed, but thought it might be causing a conflict, so it is disabled, but still not sub-text.

I'm using v 3.4.1 of the template on Joomla 2.5.7

Site: www.portlandrfc.com

Thanks

**Comment by dziudek** [comment]
Hi,

These texts are used in the description property in the GK ExtMenu plugin which adds additional tab with settings for every menu item - so please install this plugin and then use this feature.

**Comment by tlovering** [comment]
Thanks so much for the tip. I'm having trouble finding that plugin as a stand alone download. The documentation I've found suggests that it's already integrated into many of the Gavick themes, although it don't see it as part of Music Free. 
The Menu options in the Template Manager don't have the same options available as those illustrated in the documentation I found here: http://www.gavick.com/documentation/joomla-templates/templates-for-joomla-1-6/gavern-3-0/gavern-3-0-menu/
